src/docfx/lib/process/InterProcessReaderWriterLock.cs

```diff
@@ -51,8 +51,10 @@ private static FileStream WaitFile(string name, string lockHash, FileAccess acce
                 }
                 catch
                 {
-                    if (DateTime.UtcNow - start > TimeSpan.FromSeconds(30))
+                    var now = DateTime.UtcNow;
+                    if (now - lastPrintTime > TimeSpan.FromSeconds(30))
                     {
+                        lastPrintTime = now;
```

Comment:
This avoid printing too much `Waiting for another process to access` to the console.